ocs_ci/ocs/utils.py

```diff
@@ -1266,3 +1271,39 @@ def get_primary_cluster_config():
     for cluster in ocsci_config.clusters:
         if cluster.MULTICLUSTER["primary_cluster"]:
             return cluster
+
+
+def collect_pod_container_rpm_package(dir_name):
+    """
+    Collect information about rpm packages from all containers from
+
+    Args:
+        dir_name(str): directory to store container rpm package info
+
+    """
+    from ocs_ci.ocs.resources import pod
+
+    create_directory_path(dir_name)
+    log.info(f"Directory path for rpm logs is{dir_name}")
+    pods = pod.get_all_pods(namespace=defaults.ROOK_CLUSTER_NAMESPACE)
```

Comment:
Please rather then defaults.ROOK_CLUSTER_NAMESPACE use:
namespace = ocsci_config.ENV_DATA["cluster_namespace"]

And use this namespace in every place where you are bellow using it from defaults.